programs/server/users.xml

```diff
@@ -104,6 +104,9 @@
             <profile>default</profile>
             <quota>default</quota>
         </ghost>
+        <system>
```

Comment:
Can we remove `ghost` user ?

Reply:
I see that ghost users are only useful during testing, and they have been configured during testing, so we can be deleted here